samples/bluetooth/central_hids/src/main.c

```diff
@@ -109,10 +110,11 @@ static void scan_filter_no_match(struct bt_scan_device_info *device_info,
 		printk("Direct advertising received from %s\n", addr);
 		bt_scan_stop();
 
-		conn = bt_conn_create_le(device_info->addr,
-					 device_info->conn_param);
+		rc = bt_conn_le_create(device_info->addr,
+				       BT_CONN_LE_CREATE_CONN,
+				       device_info->conn_param, &conn);
 
-		if (conn) {
+		if (!rc && conn) {
```

Comment:
Checking `rc` should be enough, also I'd prefer `err` which is consistent with the rest of the file.

Reply:
done